frontend/database/103_groups_identities_make_accept_teacher_boolean.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE `Groups_Identities`
+  CHANGE COLUMN `accept_teacher` `accept_t` enum('yes','no');
+
+ALTER TABLE `Groups_Identities`
+  ADD `accept_teacher` tinyint(1) DEFAULT NULL COMMENT 'Almacena la respuesta del participante de un curso si acepta al organizador como su maestro.';
+
+UPDATE `Groups_Identities`
+  SET `accept_teacher` = 1
```

Comment:
```suggestion
  SET `accept_teacher` = (`accept_t` = 'yes');
```

funciona? si sí, no necesitas L11-13.

Reply:
Al menos ningún script se ha quejado cuando ejecuté el `sql`